src/htmlminifier.js

```diff
@@ -615,119 +615,101 @@
         };
       }
     }
-  }
 
-  function minifyURLs(text, options) {
-    if (typeof options !== 'object') {
-      options = { };
+    if (options.minifyURLs && typeof options.minifyURLs !== 'object') {
+      options.minifyURLs = { };
     }
 
-    try {
-      // try to get global reference first
-      var __RelateUrl = global.RelateUrl;
-
-      if (typeof __RelateUrl === 'undefined' && typeof require === 'function') {
-        __RelateUrl = require('relateurl');
-      }
-
-      // noop
-      if (!__RelateUrl) {
-        return text;
+    if (options.minifyJS) {
+      if (typeof options.minifyJS !== 'object') {
+        options.minifyJS = { };
       }
+      options.minifyJS.fromString = true;
+      (options.minifyJS.output || (options.minifyJS.output = { })).inline_script = true;
+    }
 
-      if (__RelateUrl.relate) {
-        return __RelateUrl.relate(text, options);
+    if (options.minifyCSS) {
+      if (typeof options.minifyCSS !== 'object') {
+        options.minifyCSS = { };
       }
-      else {
-        return text;
+      if (typeof options.minifyCSS.advanced === 'undefined') {
+        options.minifyCSS.advanced = false;
       }
     }
-    catch (err) {
-      log(err);
-    }
+  }
+
+  function noop(text) {
     return text;
   }
 
-  function minifyJS(text, options) {
-    if (typeof options !== 'object') {
-      options = { };
+  function getModule(name, path) {
+    // try to get global reference first
+    var result = global[name];
+    if (typeof result === 'undefined' && typeof require === 'function') {
+      result = require(path);
```

Comment:
That doesn't play well with webpack:

> WARNING in ./~/html-minifier/src/htmlminifier.js
Critical dependencies:
649:15-28 the request of a dependency is an expression
 @ ./~/html-minifier/src/htmlminifier.js 649:15-28

And later

> webpack:///./~/html-minifier/src ^\.\/.*$:13
	return map[req] || (function() { throw new Error("Cannot find module '" + req + "'.") }());
                                 ^
Error: Cannot find module 'relateurl'.